docker/clear.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+docker rm $(docker ps -a -q -f status=exited)
```

Comment:
yeah, I can read man for it
But it's not intuitive why you read this code w/o lurking